src/main/webapp/app/exercises/shared/result/result-detail.component.html

```diff
@@ -24,7 +24,21 @@ <h6 [innerHTML]="'artemisApp.result.afterDueDateFeedbackHidden' | artemisTransla
         <div *ngIf="!loadingFailed && filteredFeedbackList && filteredFeedbackList.length; else buildLogsContainer" class="result-detail-container">
             <div *ngIf="showScoreChart && result.participation?.exercise" class="result-score-chart">
                 <span class="result-score-chart-title">
-                    <span jhiTranslate="artemisApp.result.score">Score</span>: {{ roundScoreSpecifiedByCourseSettings(result.score, course) }}%
+                    <span *ngIf="!exercise || !exercise.maxPoints; else scoreWithPoints">
+                        {{ 'artemisApp.result.score' | artemisTranslate: { score: roundValueSpecifiedByCourseSettings(result.score, course) } }}
+                    </span>
+                    <ng-template #scoreWithPoints>
+                        <span *ngIf="result.score && exercise && exercise.maxPoints">
+                            {{
```

Comment:
Formatting intentional? Seems a bit weird.

Reply:
Since this line is so long, this was the nicest formatting I found